packages-web/datagrid/src/components/__tests__/Pagination.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import { shallow } from "enzyme";
+import { createElement } from "react";
+import { ClientSidePagination, ClientSidePaginationProps, Pagination, PaginationProps } from "../Pagination";
+
+describe("Pagination", () => {
+    it("renders the structure correctly", () => {
```

Comment:
maybe check that first/previous/next buttons are disabled at the right time

Reply:
Done